spec/cvss_suite_spec.rb

```diff
@@ -16,6 +16,7 @@
   end
 
   it 'is invalid' do
-    expect{CvssSuite.new('Not a valid vector!')}.to raise_error(RuntimeError, 'Vector is not valid!')
+    expect { CvssSuite.new('Not a valid vector!') }
+      .to raise_error(CvssSuite::Errors::InvalidVector, 'Vector is not valid!')
```

Comment:
actually now that we're classing this, we don't really care about the string anymore

Reply:
yeah we’ll have a specific string in the actual error, but i don’t think the test should assert on the string itself (because it could change  and we don’t care), the error class should be sufficient